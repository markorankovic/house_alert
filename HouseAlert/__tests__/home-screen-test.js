import React from 'react'
import TestRenderer from 'react-test-renderer'
import HomeScreen from "../src/screens/home-screen"

test('Renders correctly', () => {
    const tree = TestRenderer.create(<HomeScreen />).toJSON()
    expect(tree).toMatchSnapshot()
})