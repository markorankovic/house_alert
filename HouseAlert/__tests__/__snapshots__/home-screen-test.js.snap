// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Who do you want to alert?
  </Text>
  <RCTScrollView
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "flexGrow": 0,
        "padding": 100,
      }
    }
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
